src/Abp.Web/Auditing/WebClientInfoProvider.cs

```diff
@@ -47,11 +47,12 @@ protected virtual string GetClientIpAddress()
 
             var clientIp = httpContext.Request.ServerVariables["HTTP_X_FORWARDED_FOR"] ??
                            httpContext.Request.ServerVariables["REMOTE_ADDR"];
-            
-            clientIp = clientIp.Remove(clientIp.IndexOf(':'));
 
+            
             try
             {
+                clientIp = clientIp.Contains(":") ? clientIp.Remove(clientIp.IndexOf(':')) : clientIp;
```

Comment:
Let's add a comment `// Remove port if present` above this line.

I don't think we should move this into the `try` block, since it's not expected to throw any exceptions.

Reply:
I will